aqua/misc.aqua

```diff
@@ -0,0 +1,72 @@
+import "@fluencelabs/aqua-lib/builtin.aqua"
+import "registry-service.aqua"
+import "constants.aqua"
+
+alias ResourceId: string
+alias Error: string
+
+func wait(successful: *bool, len: i16, timeout: u16) -> bool:
+  status: *string
+  waiting = (arr: *bool, s: *string):
+    join arr[len - 1]
+    s <<- "ok"
+
+  waiting(successful, status)
+  par status <- Peer.timeout(timeout, "timeout")
+
+  result: *bool
+  -- HACK: workaround for #LNG-63
+  stat = status!
+  if stat == "ok":
+    result <<- true
+  else:
+    result <<- false
+
+  <- result!
+
+func appendErrors(error1: *Error, error2: *Error):
+  for e <- error2:
+    error1 <<- e
+
+func getResourceId(label: string, peer_id: string) -> ResourceId:
```

Comment:
What's the current public function to calculate resource id?

Also, is it possible to do locally on JS Peer, without going to Rust Peer?

Reply:
Yeah, I should move it to public API and now is not possible to calculate resource id without Registry.